documentation/single-file-tools.md

```diff
@@ -0,0 +1,67 @@
+  - The workload is being run in a container with the runtime available. This could be either because you are running a workload on our [official .NET runtime containers](https://hub.docker.com/_/microsoft-dotnet-runtime) or you built a custom image that installs it at build time.
+- These tools are bundles that will inflate on the target machine and will extract files to disk as necessary. The extraction folder is by default within the `TEMP` directory. In case `TEMP` is not available in the target environment or you want to control such directory for reasons such as clean up, please set `DOTNET_BUNDLE_EXTRACT_BASE_DIR` to a path that can be used in lieu of `TEMP`.
+
+## Obtaining the Tools
+
+### Downloading the latest available release of a tool
+
+The latest release of our tools is always available at a link that follows the following schema:
+
+```
+https://aka.ms/<TOOL NAME>/<TARGET PLATFORM RUNTIME IDENTIFIER>
+```
+
+The tools we support are:
+- `dotnet-counters`
+- `dotnet-dump`
+- `dotnet-gcdump`
+- `dotnet-sos`
+- `dotnet-trace` 
+  
+The supported [runtime identifiers](https://docs.microsoft.com/en-us/dotnet/core/rid-catalog) are:
+
+- `win-arm`
+- `win-arm64`
+- `win-x64`
+- `win-x86`
+- `osx-x64`
+- `linux-arm`.
+- `linux-arm64`
+- `linux-musl-arm64`
+- `linux-x64`
+- `linux-musl-x64`
+
+For example, the latest release of `dotnet-trace` for Windows x64 will be available at https://aka.ms/dotnet-dump/win-x64.
+
+To download these tools you can either use a browser or use a command line utility like `wget` or `curl`. It's important to note that the file name is returned in the `content-disposition` header of the request. If the utility you use to download the tools respects the header the file will be saved with the name of the tool downloaded, otherwise you might need to rename the tool. A few examples of ways to preserve the name:
+
+1. *`curl` (available on Windows after 1706, macOS, and several distributions of Linux)*
+```sh
+curl -JLO https://aka.ms/dotnet-dump/win-x64
+```
+
+2. *`wget`*
+```sh
+wget --content-disposition https://aka.ms/dotnet-dump/linux-x64
+```
+
+3. *`pwsh` (PowerShell core)*
+```powershell
+$resp = Invoke-WebRequest -Uri "https://aka.ms/dotnet-dump/win-x86"
+$header = [System.Net.Http.Headers.ContentDispositionHeaderValue]::Parse($resp.Headers.'content-disposition')
+[System.IO.File]::WriteAllBytes($header.FileName, $resp.content)
+```
+
+### Past Releases and Checksum Validation
+
+Each release in the [releases section](https://github.com/dotnet/diagnostics/releases) of the repository contains a table of stable links for our tools starting release `v5.0.152202`.  Additionally, there's a CSV available as an attachment containing all the stable links and SHA512 checksums in case the downloaded file needs to be validated.
```

Comment:
```suggestion
Each release in the [releases section](https://github.com/dotnet/diagnostics/releases) of the repository contains a table of stable links for our tools starting release `v5.0.152202`. Additionally, there's a CSV available as an attachment containing all the stable links and SHA512 checksums in case the downloaded file needs to be validated.
```